frontend/src/components/table/row.tsx

```diff
@@ -0,0 +1,65 @@
+import { FaEdit, FaTrash } from 'react-icons/fa';
+import {
+  FileDataTypes,
+  FunctionDataTypes,
+  ShelfDataTypes,
+} from '../../interfaces/dataTypes';
+import CheckBox from '../UI/checkbox/checkBox';
+import { StyledRow, StyledCell, ActionContainer } from './table -styles';
+
+interface RowProps {
+  data: FunctionDataTypes | FileDataTypes | ShelfDataTypes;
+  multiSelect?: boolean;
+  selectedRows: (FunctionDataTypes | FileDataTypes | ShelfDataTypes)[];
+  setSelectedRows: (
+    data: (FunctionDataTypes | FileDataTypes | ShelfDataTypes)[]
+  ) => void;
+  isChecked?: boolean;
+}
+
+export const Row = ({
```

Comment:
Is this just any "row" or a specific "row" with specific use?

Reply:
Will write a better name for that component.